ui/serializers.py

```diff
@@ -67,8 +71,5 @@ class DropboxFileSerializer(serializers.Serializer):
     isDir = serializers.BooleanField()
 
     def create(self, validated_data):
-        video, _ = Video.objects.get_or_create(
-            s3_object_key=validated_data["name"],
-            defaults={'source_url': validated_data["link"]},
-        )
+        video, _ = Video.objects.get_or_create(defaults={'source_url': validated_data["link"]},)
```

Comment:
The Video object no longer has an 's3_object_key', that has been moved to VideoFile model.  But this create method isn't used anywhere, maybe I should remove it altogether.